src/cppcache/src/CppCacheLibrary.cpp

```diff
@@ -127,15 +127,18 @@ std::string CppCacheLibrary::getProductLibDir() {
   for (int i = 0; i < PATH_MAX && path[i] != 0; i++) {
     path[i] = ::tolower(path[i]);
   }
-  dllNamePtr = strstr(path, "apache-geode.dll");
+  std::string cppName = PRODUCT_LIB_NAME;
+  cppName += ".dll";
+  std::string dotNetName = PRODUCT_DLL_NAME;
+  dotNetName += ".dll";
+  dllNamePtr = strstr(path, cppName.c_str());
```

Comment:
Nit: `strstr` seems old-school.  If you turn `path` into a `std::string` then you can use `std::basic_string::find` and do something like.

```cpp
if (pathAsString.find(cppName) == std::string::npos)
{
    // ...
}
```